server/app/utils.py

```diff
@@ -49,6 +50,8 @@ def parse_date(date):
         date = datetime.datetime.strptime(
             date, "%Y-%m-%d %H:%M:%S")
 
+    logging.debug("utils.parse_date being used")
```

Comment:
I'm not sure if this method is called anywhere - it doesn't look like it - but it is imported in places.